websites/M/Maliki/iframe.ts

```diff
@@ -0,0 +1,10 @@
+const iframe = new iFrame();
+
+iframe.on("UpdateData", async () => {
+	if (document.querySelector(".artiste") && document.querySelector(".titre")) {
```

Comment:
make these constants to avoid computing them twice

Reply:
Done in commit 21851275d7a887dfd0a995e019632c902ce63e77